### PR TITLE
Implement LIC 12

### LINKED ISSUE #9: Implement LIC 12
Description from the assignment:

```
There exists at least one set of two data points, separated by exactly K_PTS consecutive
intervening points, which are a distance greater than the length, LENGTH1, apart. In addition, there exists at least one set of two data points (which can be the same or different from
the two data points just mentioned), separated by exactly K_PTS consecutive intervening
points, that are a distance less than the length, LENGTH2, apart. Both parts must be true
for the LIC to be true. The condition is not met when NUMPOINTS < 3.

0 ≤ LENGTH2
```

### PR DESCRIPTION
**Pull request by K2017**
Fixes #9

### CODE REVIEW

File: src/test/java/LIC12Test.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public final class LIC12Test {
+  Parameters parameters = new Parameters();
+
+  @Test
+  public void LIC12InRange() {
+    var points = new Point[] {
```
Comment (andreaskth) [inline src/test/java/LIC12Test.java:10]: This test needs a comment in natural language describing it

File: src/test/java/LIC12Test.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    CMV cmv = new CMV(parameters, points);
+    cmv.populate();
+
+    assertTrue(cmv.get()[12]);
+  }
+
+  @Test
+  public void LIC12OutOfRange() {
+    var points = new Point[] {
```
Comment (andreaskth) [inline src/test/java/LIC12Test.java:29]: This one also needs descriptive comment

**Review by andreaskth — CHANGES_REQUESTED** [review]
Apart from lacking comments describing tests it seems to be "all good" as far as I can see. Good job!

**Review by elmaxe — APPROVED** [review]
Needs comments on the tests

**Review by elmaxe — APPROVED** [review]

**Review by andreaskth — APPROVED** [review]
Looks good now!

**Pull request merged**